liquibase/liquibase
# Do not worry about FileSystemAlreadyExistsException errors

Fixes #1486

LB-832

- - -

## Dev Handoff Notes (Internal Use)

#### Links

- Fixed Issue: https://github.com/liquibase/liquibase/issues/1486
- Local Branch: https://github.com/liquibase/liquibase/tree/LB-832
- Unit Tests: https://github.com/liquibase/liquibase/pull/1885/checks
- Integration Tests: (shown in Unit Tests link above)
- Functional Tests: https://jenkins.datical.net/job/liquibase-pro/job/LB-832/

#### Testing

- Steps to Reproduce: https://github.com/liquibase/liquibase/issues/1486
- Guidance: 
  - Only added handling for `FileSystemAlreadyExistsException`
  
#### Dev Verification

Replicated it with maven to ensure the error was no longer showing up
(No output was saved at the time)

## Test Requirements - Liquibase Internal QA
* No new functional automated test required for this issue.

This particular error shows up in a variety of circumstances. Internal Liquibase QA can reproduce the issue using Liquibase Maven.  Manual testing is for that specific use case. 

_Setup_


* Execute a mvn clean
* Verify that the /target directory no longer exists in your Maven project.

_Verify mvn liquibase:update does not issue a WARNING that target/test-classes cannot be found._


* The WARNING is now INFO

_Verify mvn liquibase:update does not expose a stack trace in the console due to missing target/test-classes._



┆Issue is synchronized with this [Jira Story](https://datical.atlassian.net/browse/LB-1786) by [Unito](https://www.unito.io)
